x-pack/plugins/console_extensions/index.js

```diff
@@ -13,7 +13,8 @@ export function consoleExtensions(kibana) {
       return (
         config.get('console_extensions.enabled') &&
         config.has('console.enabled') &&
-        config.get('console.enabled')
+        config.get('console.enabled') &&
+        !config.get('elasticsearch.tribe.url')
```

Comment:
tested with both `node scripts/kibana` and `node scripts/kibana --elasticsearch.tribe.url http://localhost:9200